go/cmd/vtctldclient/plugin_grpcvtctlclient.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2020 The Vitess Authors.
```

Comment:
Tiny nit: copyright is the wrong year

Reply:
More like copywrong, amirite

Reply:
ahah i copied this file from `vtctlclient`